configure.ac

```diff
@@ -7707,13 +7715,16 @@ AX_OUT_OF_TREE_FILE([wolfssl/wolfcrypt/port/intel/quickassist_mem.h])
 AC_OUTPUT
 
 
-# force make clean
-AC_MSG_NOTICE([---])
-AC_MSG_NOTICE([Running make clean...])
-if test "$verbose" = "yes"; then
-    make clean
-else
-    make clean >/dev/null
+if test "$ENABLED_MAKECLEAN" = "yes"
+then
+    # force make clean
+    AC_MSG_NOTICE([---])
+    AC_MSG_NOTICE([Running make clean...])
+    if test "$verbose" = "yes"; then
+        make clean
```

Comment:
Can you also look into changing make to use an environment variable`$(MAKE)`? Or are you planning a new PR for that.

Reply:
Sure, I will confirm that works.